### ISSUE TITLE
Cursor leaves a bunch of copies of itself when moving around (osx)

### ISSUE THREAD

**Issue by djhunter** [otfried/ipe#34]
Usually after selecting a new tool with grid snapping on, the first time I use the tool the cursor doesn't erase itself as it moves around the screen. (see attached screenshot) Happens with other objects too (like the whole image when panning the canvas).
Ipe version 7.2.1
OSX version 10.11.2 (El Capitan)

![screen shot 2016-01-08 at 1 43 34 pm](https://cloud.githubusercontent.com/assets/4453085/12210310/df2c6160-b60d-11e5-887b-ee12e257d66c.png)

**Comment by m7thon** [comment]
I observed this, too. But for me this does not happen from the start of ipe, but only after clicking through several draw modes. When this happens, ipe also closes without asking to save.

I did not further test this, since this (or some other problem) caused my Finder to crash so badly that I needed to force my laptop off by holding the power button to resolve.

**Comment by otfried** [comment]
I don't see how anything that Ipe does could cause your Finder to crash.  Did you start Ipe from the Finder or from the command line?   Is this using my Ipe.app bundle or compiling yourself?

**Comment by m7thon** [comment]
I used the Ipe.app provided on your homepage, moved to /Applications and started from there. I was able to crash ipe by inserting a jpeg (not reproducibly, though), at some point later a file dialog (open or save, I don't remember) looked strange (one filename entry shifted from its row), and later (after closing Ipe) my Finder no longer opened and gave a spinning ball instead. Restarting (via "Force quit") did not work. My computer would also no longer shut down cleanly.

I am aware this is a strange report and does not really help you at all, but it was somehow related to the current Ipe.app (I even worried if the download site you use is trustworthy...). I am somewhat hesitant to further investigate this.
